internal/store/model/assessment.go

```diff
@@ -24,10 +22,11 @@ type Assessment struct {
 }
 
 type Snapshot struct {
-	ID           uint                      `gorm:"primaryKey;autoIncrement"`
-	CreatedAt    time.Time                 `gorm:"not null;default:now()"`
-	Inventory    *JSONField[api.Inventory] `gorm:"type:jsonb;not null"`
-	AssessmentID uuid.UUID                 `gorm:"not null;type:VARCHAR(255);"`
+	ID           uint      `gorm:"primaryKey;autoIncrement"`
+	CreatedAt    time.Time `gorm:"not null;default:now()"`
+	Version      uint      `gorm:"not null;default:1;type:SMALLINT"`
```

Comment:
I don't think this change is related to this PR
Adding new version field to the snapshot
Am I right?
Avoiding this change would remove the DB migration change, so I think better to remove it to the related PR